__tests__/app.tests.js

```diff
@@ -0,0 +1,37 @@
+const app = require('../app');
+const request = require('supertest');
+const data = require('../db/data/test-data');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection.js');
+
+afterAll(()=>{
+    return db.end();
+})
+beforeEach(()=>{
+    return seed(data)
+})
+
+describe('app',()=>{
+    test('responds with status of 200 and object with description and slug keys',()=>{
+        return request(app).get('/api/topics').expect(200)
+        .then((response)=>{
+            const {topics} = response.body
+            topics.forEach((topic)=>{
```

Comment:
I'd like to see an array length check here please Sula as well as the `.forEach`